projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -69,51 +82,71 @@ function goToProductFromSearch(id: string, mobile: boolean) {
   }
 }
 
-export function addProductToCartViaAutoComplete(mobile: boolean) {
-  const product = products[0];
+function checkMiniCartCount(expectedCount) {
+  return cy.get('cx-mini-cart').within(() => {
+    cy.get('.count').should('contain', expectedCount);
+  });
+}
 
-  goToProductFromSearch(product.code, mobile);
+export function validateEmptyCart() {
+  cy.get('cx-breadcrumb h1').should('contain', 'Your Shopping Cart');
+  cy.get('.EmptyCartMiddleContent').should(
+    'contain',
+    'Your shopping cart is empty'
+  );
+}
 
+export function addToCart() {
   cy.get('cx-add-to-cart')
     .getAllByText(/Add To Cart/i)
     .first()
     .click({ force: true });
-  cy.get('cx-added-to-cart-dialog [aria-label="Close"]').click({ force: true });
+}
 
-  const miniCart = cy.get('cx-mini-cart');
-  miniCart.within(() => {
-    cy.get('.count').should('contain', 1);
-  });
-  miniCart.click({ force: true });
+export function closeAddedToCartDialog() {
+  cy.get('cx-added-to-cart-dialog [aria-label="Close"]').click({ force: true });
+}
 
-  getCartItem(product.name).within(() => {
+export function checkProductInCart(product, qty = 1) {
+  return getCartItem(product.name).within(() => {
     cy.get('.cx-price>.cx-value').should('contain', formatPrice(product.price));
-    cy.get('.cx-counter-value').should('have.value', '1');
-    cy.get('.cx-total>.cx-value').should('contain', formatPrice(product.price));
+    cy.get('.cx-counter-value').should('have.value', `${qty}`);
+    cy.get('.cx-total>.cx-value').should(
+      'contain',
+      formatPrice(qty * product.price)
+    );
+    cy.root();
   });
 }
 
+export function checkAddedToCartDialog(itemsNumber = 1) {
+  cy.get('cx-added-to-cart-dialog .cx-dialog-total').should(
+    'contain',
+    `Cart total (${itemsNumber} item${itemsNumber > 1 ? 's' : ''})`
+  );
+}
+
+export function addProductToCartViaAutoComplete(mobile: boolean) {
```

Comment:
You can as a PR author add your own comments to initiate discussion around code that you have some doubts about.